sync-core/src/test/java/com/cloudant/mazha/SpecifiedCouch.java

```diff
@@ -14,6 +14,9 @@
 
 package com.cloudant.mazha;
 
+import com.cloudant.sync.util.Misc;
```

Comment:
Looks like you can revert this whole file as it only adds unused imports and tweaks whitespace